src/server/scripts/Outland/zone_shadowmoon_valley.cpp

```diff
@@ -1635,6 +1635,71 @@ class npc_shadowmoon_tuber_node : public CreatureScript
     }
 };
 
+enum KorWild
+{
+    SAY_LAND    = 0,
+    POINT_LAND  = 1
+};
+
+class npc_korkron_or_wildhammer : public ScriptedAI
+{
+public:
+    npc_korkron_or_wildhammer(Creature* creature) : ScriptedAI(creature)
+    {
+        creature->SetDisableGravity(true);
+        creature->SetHover(true);
+    }
+
+    void Reset() override
+    {
+        me->SetReactState(REACT_PASSIVE);
+        me->SetUnitFlag(UNIT_FLAG_NOT_SELECTABLE | UNIT_FLAG_NON_ATTACKABLE);
+    }
+
+    void JustDied(Unit* /*killer*/) override
+    {
+        me->DespawnOrUnsummon(3s, 0s);
+    }
+
+    void IsSummonedBy(WorldObject* summoner) override
+    {
+        PlayerGUID = summoner->GetGUID();
+        me->SetFacingToObject(summoner);
+        Position pos = summoner->GetPosition();
+        me->GetMotionMaster()->MovePoint(POINT_LAND, pos);
+    }
+
+    void MovementInform(uint32 type, uint32 id) override
+    {
+        if (type == POINT_MOTION_TYPE && id == POINT_LAND)
+        {
+            if (Player* player = ObjectAccessor::GetPlayer(*me, PlayerGUID))
```

Comment:
```suggestion
            if (Player* player = ObjectAccessor::GetPlayer(*me, _playerGUID))
```

Reply:
?

Reply:
Change the name of that variable to be properly identified as private, and along with that change the P to p as capitalized names are used only for naming classes and not variables.